tests/validate/ps_bandwidth_test/src/host.cpp

```diff
@@ -72,19 +72,18 @@ int main(int argc, char** argv) {
         return EXIT_FAILURE;
     }
 
-    std::cout << "\nNOT SUPPORTED" << std::endl;
-    return EOPNOTSUPP;
 
     auto num_devices = xrt::system::enumerate_devices();
 
     auto device = xrt::device {dev_id};
 
     // Load dependency xclbins onto device if any
     for (const auto& path : dependency_paths) {
-        auto retVal = validate_binary_file(path);
+        auto x_file = test_path + "/" + path;
```

Comment:
To make this work with XRT we need to update how SubCmdValidate works but this makes more sense than what I did before :)